subgraphs/stakedao/src/mappings/strategyMappings.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  VaultFee,
+  Vault as VaultStore,
+  Strategy as StrategyStore,
+} from "../../generated/schema";
+import {
+  SetWithdrawalFeeCall,
+  SetPerformanceFeeCall,
+  Harvested as HarvestedEvent,
+} from "../../generated/templates/Strategy/Strategy";
+
+import { BigInt, log } from "@graphprotocol/graph-ts";
+
+
+export function handleHarvested(event: HarvestedEvent): void {
+  const strategyAddress = event.address;
+  const strategy = StrategyStore.load(strategyAddress.toHexString());
+
+  if (strategy) {
+    const vaultAddress = strategy.vaultAddress;
+    const vault = VaultStore.load(vaultAddress.toHexString());
+
+    // load performance fee and get the fees percentage
+    const vaultFee = VaultFee.load(vault!.fees[1]);

```

Comment:
It's better if you loop through to find performance fee instead of assuming performance fee is always index ` in the array